src/hoc_components/FixedDataTableRowLegacyFunction.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import FixedDataTableTranslateDOMPosition from '../FixedDataTableTranslateDOMPosition';
+import cx from '../vendor_upstream/stubs/cx';
+import joinClasses from '../vendor_upstream/core/joinClasses';
+
+function RowLegacy(props) {
+  const subRowHeight = props.subRowHeight || 0;
+  let style_wrapper = {
```

Comment:
We can change the properties of const object as well,  use `const` wherever you use `let` just because you later want to modify some of its properties.

Reply:
Done